src/main/resources/assets/tconstruct/book/encyclopedia/zh_cn/upgrades/general/tconstruct_smelting.json

```
@@ -0,0 +1,16 @@
+{
+  "modifier_id": "tconstruct:smelting",
+  "text": [
+    {
+      "text": "当工具造成伤害、挖掘方块、发射箭矢或格挡伤害时烧炼其中物品。单次烧炼1个物品。"
```

```suggestion
      "text": "当工具造成伤害、挖掘方块、发射箭矢或格挡伤害时烧炼工具中的物品。单次烧炼1个物品。"
```
The original usage of "其"(their) may be ambiguous.
Recommended but not required.